package.json

```diff
@@ -10,6 +10,7 @@
     "build": "loom build",
     "build-consumer": "yarnpkg build && ./scripts/build-consumer.sh",
     "build-consumer-spin": "yarnpkg build && ./scripts/build-consumer-spin.sh",
+    "build:api-docs": "yarn workspace @shopify/checkout-ui-extensions build-docs",
```

Comment:
I recommend taking over [yarn docs:checkout](https://github.com/Shopify/ui-extensions/blob/8c22bceacf7fb1a498cb7a9b54cf69f4910eed4c/package.json#L18). You should rename the old one to `yarn deprecated:docs:checkout`. We need to keep using that old command until the [new generator](https://github.com/Shopify/checkout-web/issues/18175) works.